logdna/common_test.go

```diff
@@ -106,15 +106,19 @@ func fmtTestConfigResource(objTyp, rsName string, pcArgs []string, rsArgs map[st
 }
 
 func fmtProviderBlock(args ...string) string {
-	opts := []string{serviceKey, ""}
+	opts := []string{serviceKey, "", "regular"}
 	copy(opts, args)
-	sk, ul := opts[0], opts[1]
+	sk, ul, tp := opts[0], opts[1], opts[2]
```

Comment:
What do you think of renaming these variables to something that imparts meaning? From the following `fmt.Sprintf`, it looks like `sk` => `serviceKey` would make sense. I have no idea what the other two variables would be (the crux of the issue).